tests/pw/tests/e2e/productAddons.spec.ts

```
@@ -1,37 +1,47 @@
 import { test, request, Page } from '@playwright/test';
 import { ProductAddonsPage } from '@pages/productAddonsPage';
+import { ProductsPage } from '@pages/productsPage';
 import { ApiUtils } from '@utils/apiUtils';
 import { data } from '@utils/testData';
 import { payloads } from '@utils/payloads';
 import { dbUtils } from '@utils/dbUtils';
 import { serialize } from 'php-serialize';
 import { responseBody } from '@utils/interfaces';
+import { serialize } from 'php-serialize';
```

Biome

[error] Shouldn't redeclare 'serialize'. Consider to delete it or rename it.

'serialize' is defined here:


(lint/suspicious/noRedeclare)